### PR TITLE
Completed html-forms

### PR DESCRIPTION
**Pull request by ayshahussain141**
Html forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        <label for="HTML">HTML</label>
+        <input type="radio" id="CSS" name="languages" value="CSS">
+        <label for="CSS">CSS</label>
+        <input type="radio" id="JavaScript" name="languages" value="JavaScript" >
+        <label for="JavaScript">JavaScript</label>
+      </div>
+
+      <div>
+        <p>Would you like to sign up for more information?</p>
+        <input type="checkbox" name="service">
```
Comment (Zhiwutian) [inline html-forms/index.html:37]: don't forget the id attribute here

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        <label for="service">Sign me up!</label>
+      </div>
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <textarea></textarea>
```
Comment (Zhiwutian) [inline html-forms/index.html:43]: you will need the matching `id` attribute on this element for the label to work properly

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a few updates are needed.

**Comment by ayshahussain141** [comment]
Html forms

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
the id should be on the `textarea` not the span :)

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**